recipe/meta.yaml

```
@@ -8,23 +8,26 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 8f1747121598e06a1ea2d00df16d332cc284cdd4596bdc136b490a2122d3aa91
+  sha256: 9121d1cf320f7f32490d893340fd33117bda0a0270eb8282dfd52ae5fd3e1af6
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # torchvision is not available on s390x
+  skip: True  # [s390x]
```

```suggestion
  skip: True  # [py<37 or s390x]
```
https://inspector.pypi.io/project/timm/0.9.12/packages/01/a5/eeb717242343d9ca34e7de554a6c08d96a0cfc7005ece4f847b1753581a6/timm-0.9.12-py3-none-any.whl/timm-0.9.12.dist-info/METADATA#line.22